### PR TITLE
Create universite-du-quebec-a-montreal-prenoms.csl

### PR DESCRIPTION
**Pull request by jjrondeau**
Hi,
This new style is based on the universite-du-quebec-a-montreal.csl. The sole modification is that all author first names (author, editor, translator, etc.) are not initialized in order to give credit to women authors.

### CODE REVIEW

File: universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl [hunk 1]
```diff
@@ -0,0 +1,935 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="3" et-al-use-first="1" initialize="false" demote-non-dropping-particle="never" default-locale="fr-CA" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Université du Québec à Montréal - APA - prénoms - Études féministes - IREF (Français - Canada)</title>
+    <title-short>UQAM prénoms</title-short>
+    <id>http://www.zotero.org/styles/universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref</id>
+    <link href="http://www.zotero.org/styles/universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
```
Comment (denismaier) [inline universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl:8]: You've mentioned that this is an almost exact copy of another style. Please add another template node.

**Review by denismaier — CHANGES_REQUESTED** [review]

File: universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl (hunk 1 above)
Comment (denismaier) [reply universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl:8]: Yes, go with @bwiernik's suggestions. That's clearer.

Comment (jjrondeau) [reply universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl:8]: I try to delete line 8 in the Edit mode, but the "Commit changes" button is not active. I am not sure how I can save my commit.

Comment (denismaier) [reply universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl:8]: Are you working online or locally?

Comment (jjrondeau) [reply universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl:8]: I work online (as I usually do).

Comment (jjrondeau) [reply universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl]: It works today on my computer at work. It was not working at home.

**Comment by jjrondeau** [comment]
Hi,
I have done requested changes. Should I do something else that is missing? Thanks.

**Comment by adam3smith** [comment]
Sorry for the delay here -- style is fine. But just to be clear, the university still uses both styles?

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**